app/models/app_setting.rb

```diff
@@ -16,13 +16,19 @@ class AppSetting < ApplicationRecord
 
   after_commit :apply_settings, on: :update
 
+  def self.refresh_instance
+    @instance = nil
```

Comment:
Would `clear_cache` be a better name for this method?

Reply:
https://github.com/hpi-swt2/vm-portal/pull/481/commits/1713c798863ae4602c06abbacf66848ac00a7a7f